### PR TITLE
main/mariadb: upgrade to 10.4.6

### PR DESCRIPTION
**Pull request by J0WI**
https://mariadb.com/kb/en/library/changes-improvements-in-mariadb-104/

### CODE REVIEW

File: main/mariadb/APKBUILD [hunk 1]
```diff
@@ -184,7 +183,6 @@ check() {
 	# exclude test-connect which seems to be buggy. testsuite does not set port env var
 	# ma_test_loghandler_readonly seems to fail if /etc/my.conf.d is missing
 	ctest -E '(test-connect|ma_test_loghandler_readonly)'
```
Comment (TBK) [inline main/mariadb/APKBUILD]:
```suggestion
	ctest -E 'test-connect'
```
I also worked on upgrading to 10.4.6 and the test ma_test_loghandler_readonly passes on my machine (x86_64).

Comment (J0WI) [reply main/mariadb/APKBUILD]: Thanks, added

**Comment by andypost** [comment]
Meantime it is interesting host many packages can work with newly added features, I seen some issues because of new system tables and other changes
So it needs to check lots of apps

**Comment by ncopa** [comment]
179bcc61994dd6f23dcb44f7f2cdb5a7b0b3bc68  456e3a801ed0b427cd5c2aa73bedb6b944a5d404

**Pull request closed without merging**